src/r5py/util/memory_footprint.py

```diff
@@ -124,20 +124,20 @@ def _get_max_memory(max_memory):
     else:
         # convert to MiB
         if unit is None:
-            value *= 2**-20
+            value = value >> 20
         elif unit == "K":
-            value *= 2**-10
```

Comment:
These did not return correct values, so I changed the logic to use Python shift operators (<< and >>), which is also a bit cleaner code I think.

Reply:
If we go for integer output, I think the values should be bytes (and [`jvm.py`](https://github.com/r5py/r5py/blob/main/src/r5py/util/jvm.py#L28) adapted to accept bytes values (just the `M` removed), to then read 
```
        jpype.startJVM(
            f"-Xmx{MAX_JVM_MEMORY:d}",
            "-Xcheck:jni",
            classpath=[R5_CLASSPATH],
        )
```

Reply:
Hmm, you're right. The restriction to integer is indeed quite problematic. For example `"1000 MB"` would return 0 (found out now after a bit more testing) which is definitely not what we want. Will return the logic back to what it was. The problem why I started investigating this was due to the `"M"` conversion which actually returned incorrect output (it doubled the input).